src/css.js

```diff
@@ -165,6 +165,10 @@ function getWidthOrHeight( elem, name, extra ) {
 	) + "px";
 }
 
+function isCustomProperty( value ) {
+	return /^--.*/.test( value );
```

Comment:
We usually put definitions of regexes in the beginning of the module with `r` prefix, like - https://github.com/jquery/jquery/blob/02c5e2908c2d99f191f5ec9e365ef2a3d357d1eb/src/manipulation.js#L36

Could you maybe do the same and also get rid of the `isCustomProperty` function and just use `/^--.*/.test(...)` directly?